src/itn_uptime_analyzer/app_config.go

```diff
@@ -211,14 +218,14 @@ var (
 
 	S3Output = Option {
 		updateJSON: func (log logging.EventLogger, cfg *AppConfig) {
-			if (cfg.Output.S3Bucket == "") != (cfg.Output.S3Key == "") {
+			if (cfg.Output.S3Bucket == "") && (cfg.Output.S3Key == "") {
 				log.Fatalf("Either both or neither of S3 bucket and S3 key should be set!")
```

Comment:
You should probably update the error message here, since you change the condition.